apps/test/unit/templates/ShowCodeToggleTest.js

```diff
@@ -213,6 +213,25 @@ describe('ShowCodeToggle', () => {
         expect(studioApp().showGeneratedCode).to.have.been.called;
       });
     });
+
+    describe("And studioApp() is subsequently initialized with enableShowCode turned off", () => {
```

Comment:
You just read it with the previous describe block. Saves repeating yourself. The output makes more sense:
```
  The ShowCodeToggle component
    When studioApp() has been configured with editCode turned off
      ✔ will render the button with 'Show Code' instead of 'Show Text'
      and the show code button is pressed
        ✔ will not call onDropletToggle
      ✔ will render the button with 'Show Code' instead of 'Show Text' 
        ✔ will not call onDropletToggle 
        ✔ will instead show the generated code
      And studioApp() is subsequently initialized with enableShowCode turned off
        ✔ will reflect the most recent config passed to studioApp().init()
```

So the scenario is: 
```
  The ShowCodeToggle component,
    When studioApp() has been configured with editCode turned off,
      And studioApp() is subsequently initialized with enableShowCode turned off,
        it will reflect the most recent config passed to studioApp().init()
```